src/main/java/reactor/netty/channel/FluxReceive.java

```diff
@@ -385,9 +385,19 @@ final void onInboundComplete() {
 	final void onInboundError(Throwable err) {
 		if (isCancelled() || inboundDone) {
 			if (log.isDebugEnabled()) {
-				log.warn(format(channel, "An exception has been observed post termination"), err);
+				if (AbortedException.isConnectionReset(err)) {
+					log.debug(format(channel, "An exception has been observed post termination"), err);
```

Comment:
I would maybe slightly change the message to mention `ConnectionReset`, to better differentiate the two cases in logs

Reply:
I changed the message a bit